src/components/Notifications/SubscribeDialog.tsx

```diff
@@ -0,0 +1,140 @@
+        className={`text-brandPrimary absolute top-[40px] right-[45px] transition-all duration-500 ${
+          isHovering ? "rotate-[-20deg] scale-150" : "rotate-[-30deg] scale-125"
+        }`}
+      />
+      <StarIcon
+        className={`text-secondary absolute bottom-[40px] left-[55px] transition-all duration-500 ${
+          isHovering ? "rotate-[-20deg] scale-100" : "rotate-[-30deg] scale-95"
+        }`}
+      />
+      <StarIcon
+        className={`text-secondary absolute bottom-[40px] right-[55px] transition-all duration-500 ${
+          isHovering ? "rotate-[20deg] scale-100" : "rotate-[30deg] scale-95"
+        }`}
+      />
+      <div className="mt-2 relative block h-[200px] w-[204px] mx-auto mb-[-50px]">
+        <EnvelopeTop className="absolute bottom-0" />
+        <EnvelopePaper
+          className={`absolute bottom-0 left-4 transition-all ${
+            isHovering ? "bottom-[-8px] duration-500" : "bottom-2 duration-300"
+          }`}
+        />
+        <EnvelopeBottom className="text-brandPrimary absolute bottom-0" />
+      </div>
+    </div>
+  );
+};
+
+const SubscribeDialog = ({
+  closeDialog,
+  type,
+}: {
+  closeDialog: () => void;
+  type: "root" | "vote";
+}) => {
+  const { address } = useAccount();
+  const [isHovering, setIsHovering] = useState(false);
+  const [email, setEmail] = useState<string | undefined>(undefined);
+  const { data: delegate } = useDelegate({ address: address });
+  const existingEmail = delegate?.statement.email;
+  const hasEmail = existingEmail && existingEmail !== "";
+
+  if (!address) return null;
+
+  return (
+    <div>
+      <HeroImage isHovering={isHovering} />
+      <h2 className="text-primary text-xl font-bold mt-4">
+        Get proposal updates in your inbox!
+      </h2>
+      <p className="text-secondary mt-2 font-normal">
+        Receive email notifications when proposals go live, and when the voting
+        window is ending soon.
+      </p>
+      {!hasEmail && (
+        <div className="flex flex-col gap-1 w-full mt-4">
+          <label className="text-xs font-semibold secondary">
+            Email address
+          </label>
+          <input
+            type="text"
+            className="border bg-wash border-line placeholder:text-tertiary text-primary p-2 rounded-lg w-full"
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+          />
+        </div>
+      )}
+      <div className="flex flex-col items-center gap-1 mt-4">
+        <Button
+          variant="outline"
+          className="w-full"
+          onClick={async () => {
+            localStorage.setItem(
+              `agora-email-subscriptions--${type}`,
+              "prompted"
+            );
+            await updateNotificationPreferencesForAddress(
+              address,
+              existingEmail || email,
+              {
+                wants_proposal_created_email: "prompted",
+                wants_proposal_ending_soon_email: "prompted",
```

Comment:
`prompted` represents the fact that we've asked the user if they want to opt in or out, and they decide to ignore.